main.go

```diff
@@ -4,22 +4,26 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"github.com/antonlindstrom/mesos_stats"
-	"github.com/golang/glog"
-	"github.com/prometheus/client_golang/prometheus"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"time"
+
+	"github.com/antonlindstrom/mesos_stats"
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
 	configFile  = flag.String("config.file", "", "Path to config file.")
 	metricsPath = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+)
+
+var (
 	cpuLimitVal = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "mesos_task",
-			Name:      "cpus_limit",
+			Name:      "cpu_limit",
 			Help:      "CPU share limit.",
```

Comment:
In that case, I would probably just call it `cpu_shares` (it's also called `cpu.shares` in cgroups) and clarify the docstring. Is this a completely relative number that only has meaning in comparison to another task's shares (that would be the case if it follows the cgroups `cpu.shares` semantic exactly), or is this some number between 0 and 1 or something like that?